app/src/main/java/com/sunasterisk/a14day_challenge/adapter/ProcessAdapter.kt

```diff
@@ -0,0 +1,37 @@
+package com.sunasterisk.a14day_challenge.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.sunasterisk.a14day_challenge.R
+import kotlinx.android.synthetic.main.recycler_view_process.view.*
+
+class ProcessAdapter(private val myDataSet: ArrayList<String>, private val context: Context) :
+    RecyclerView.Adapter<ProcessAdapter.ProcessViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ProcessViewHolder {
+        val view = LayoutInflater.from(parent.context)
+            .inflate(R.layout.recycler_view_process, parent, false)
+
+        return ProcessViewHolder(view)
+    }
+
+    override fun getItemCount() = myDataSet.size
+
+    override fun onBindViewHolder(holder: ProcessViewHolder, position: Int) {
+        holder.bindData(myDataSet, position, context)
+    }
+
+    class ProcessViewHolder(val view: View) : RecyclerView.ViewHolder(view) {
+        fun bindData(myDataSet: ArrayList<String>, position: Int, context: Context) {
+            view.textDay.text = myDataSet[position]
+
+            if (position == myDataSet.size - 1) {
+                view.imageTick.background =
+                    context.resources.getDrawable(R.drawable.image_not_tick, null)
+            }
```

Comment:
1. Không cần truyền position, có thể dùng adapterPosition 
2. Không cần truyền context vào, có thể lấy từ bất kì View nào cả, điển hình ở đây là itemView(trong code là view) 
3. Sử dụng setBackgroundResource 
```suggestion
            if (adapterPosition == myDataSet.size - 1) {
                view.imageTick.setBackgroundResource(R.drawable.ic_not_tick)
            }
```